proto/tendermint/farming/v1beta1/genesis.proto

```diff
@@ -38,31 +61,11 @@ message PlanRecord {
   // plan specifies the plan interface; it can be FixedAmountPlan or RatioPlan
   google.protobuf.Any plan = 1 [(gogoproto.nullable) = false];
```

Comment:
Don't we need `(cosmos_proto.accepts_interface) = "PlanI"` option for this field?